stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -996,7 +996,7 @@ public struct Set<Element : Hashable> :
   ///
   /// In the following example, the `bothNeighborsAndEmployees` set is made up
   /// of the elements that are in *both* the `employees` and `neighbors` sets.
-  /// Elements that are only in one or the other are left out of the result of
+  /// Elements that are in only one or the other are left out of the result of
```

Comment:
After reading it a couple more times, I see how "in only one or the other" is ok. Can "either" be used instead of "only"?